Sources/Filters/General/ImageMarchingSquares/index.js

```diff
@@ -26,53 +40,66 @@ function vtkImageMarchingSquares(publicAPI, model) {
   const edgeLocator = vtkEdgeLocator.newInstance();
 
   // Retrieve scalars and pixel coordinates. i-j-k is origin of pixel.
```

Comment:
please document those new parameters

Reply:
Added JSDoc comments to document all parameters.